src/mats-method-param-code/mats-method-param-code.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Get } from '@nestjs/common';
+import { ApiBadRequestResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { MatsMethodParamCodeDTO } from 'src/dto/mats-method-param-code.dto';
+import { MatsMethodParamCodeService } from './mats-method-param-code.service';
+
+@ApiTags('MATS Method Parameter Codes')
+@Controller('mats-method-param-code')
```

Comment:
should just be @Controller() with no prams. this will cause the route path to be /api/master-data-mgmt/mats-method-param-code\mats-method-param-codes becasue the routes.ts file specifies the path for each module.